openupgrade_scripts/scripts/web/17.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    web_pwa_oca_installed = env["ir.module.module"].search(
```

Comment:
The module doesn't "exist" at this level, as it has been merged. You have to detect it by any column existence or similar.